### PR TITLE
BREAKING CHANGE:  deprecating old components

### PR DESCRIPTION
**Pull request by owilliams320**
## Description
Removing the 3.x Deprecated components

### Documentation matrix of deprecated list
| Covalent Compent | NG Material Replacement |
| ----------- | ----------- |
| data table | https://material.angular.io/components/table/overview |
| expansion panels	| https://material.angular.io/components/expansion/overview |
| sidesheet content | https://material.angular.io/components/sidenav/overview |
| virtual scrolling | https://material.angular.io/cdk/scrolling/overview |
| stepper | https://material.angular.io/cdk/stepper/overview |
| loading	| https://material.angular.io/components/progress-spinner/overview |
| notifications | https://material.angular.io/components/badge/overview |
| paging | https://material.angular.io/components/paginator/overview |
| nav steps | https://material.angular.io/components/stepper/overview |
| chips autocomplete | https://material.angular.io/components/chips/overview |
| tab select | https://material.angular.io/components/tabs/overview |
| http service |	https://angular.io/api/common/http |
| http decorators | https://angular.io/api/common/http |
| query service | https://material.angular.io/cdk/layout/overview |
| toggle directive | https://material.angular.io/cdk/layout/overview |

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:lib` still works.

### CODE REVIEW

File: src/app/content/layouts/nav-list/nav-list.component.ts [hunk 1]
```diff
@@ -157,7 +156,7 @@ export class NavListComponent {
             <mat-divider [inset]="true" *ngIf="!last"></mat-divider>
           </ng-template>
         </mat-nav-list>
-        <div tdMediaToggle="gt-xs" [mediaClasses]="['push-sm']">
+        <div  >
```
Comment (JulieMarie) [inline src/app/content/layouts/nav-list/nav-list.component.ts:159]: Pickiness....get rid of the extra spaces.

File: src/app/content/layouts/nav-view/nav-view.component.ts [hunk 2]
```diff
@@ -97,7 +97,7 @@ export class NavViewComponent {
         <a mat-icon-button matTooltip="Docs" href="https://teradata.github.io/covalent/" target="_blank"><mat-icon>chrome_reader_mode</mat-icon></a>
         <a mat-icon-button matTooltip="Github" href="https://github.com/teradata/covalent" target="_blank"><mat-icon svgIcon="assets:github"></mat-icon></a>
       </div>
-      <div layout-gt-sm="row" tdMediaToggle="gt-xs" [mediaClasses]="['push-sm']">
+      <div layout-gt-sm="row"  >
```
Comment (JulieMarie) [inline src/app/content/layouts/nav-view/nav-view.component.ts:100]: Extra spaces

File: src/app/content/layouts/manage-list/manage-list.component.ts [hunk 3]
```diff
@@ -165,7 +164,7 @@ export class ManageListComponent {
               </button>
             </div>
           </mat-toolbar>
-          <div layout-gt-sm="row" tdMediaToggle="gt-xs" [mediaClasses]="['push-sm']">
+          <div layout-gt-sm="row"  >
```
Comment (JulieMarie) [inline src/app/content/layouts/manage-list/manage-list.component.ts:167]: Extra spaces

File: src/platform/flavored-markdown/flavored-markdown.component.ts [hunk 4]
```diff
@@ -331,98 +326,98 @@ export class TdFlavoredMarkdownComponent implements AfterViewInit, OnChanges {
...
-            width.min = undefined;
-          }
-          return {
-            label: col,
-            name: col.toLowerCase().trim(),
-            numeric: /^--*[ \t]*:[ \t]*$/.test(alignment[index]),
-            width,
-          };
-        });
+  // private _replaceTables(markdown: string): string {
```
Comment (JulieMarie) [inline src/platform/flavored-markdown/flavored-markdown.component.ts]: Why all this commented out code?

**Review by JulieMarie — CHANGES_REQUESTED** [review]

File: src/app/content/layouts/manage-list/manage-list.component.ts (hunk 3 above)
Comment (owilliams320) [reply src/app/content/layouts/manage-list/manage-list.component.ts:167]: doh

File: src/platform/flavored-markdown/flavored-markdown.component.ts (hunk 4 above)
Comment (owilliams320) [reply src/platform/flavored-markdown/flavored-markdown.component.ts]: ah good catch. it was replacing tables with the deprecated td data tables....will look into replacing this with mat table

**Review by JulieMarie — APPROVED** [review]
Looks good now!

**Pull request merged**